api/curve/v1/utils.py

```diff
@@ -76,7 +76,7 @@ def encode_if_unicode(s):
     :param s:
     :return:
     """
-    if isinstance(s, unicode):
+    if isinstance(s, str):
```

Comment:
File "Curve/api/curve/v1/utils.py", line 80, in encode_if_unicode
    if isinstance(s, unicode):
NameError: name 'unicode' is not defined

Fix for above error

Reply:
Thanks for your revision.